ee/tabby-webserver/src/schema/repository.rs

```diff
@@ -1,10 +1,24 @@
 use anyhow::Result;
 use async_trait::async_trait;
-use juniper::{GraphQLObject, ID};
+use juniper::{FieldError, GraphQLObject, IntoFieldError, ScalarValue, ID};
 use juniper_axum::relay::NodeType;
 
 use super::Context;
 
+#[derive(thiserror::Error, Debug)]
+pub enum RepositoryError {
```

Comment:
Please follow the definition of https://github.com/TabbyML/tabby/blob/main/ee/tabby-webserver/src/schema/auth.rs#L81 closely, thus the validation error could be passed to client side for automatic field based error message display